Jellyfin.Data/Jellyfin.Data.csproj

```diff
@@ -4,7 +4,6 @@
     <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <TreatWarningsAsErrors Condition=" '$(Configuration)' == 'Release' ">true</TreatWarningsAsErrors>
```

Comment:
Why did you remove this?

Reply:
It caused a ton of errors that were causing build failures.

Reply:
looks like all doc errors. Is there a way to disable just that warning?

Reply:
You can in the csproj file by adding `<NoWarn>1591</NoWarn> in the `PropertyGroup` tag

Reply:
I think we should merge this to fix the CI and then debate about the ideal solution in chat to avoid throwing any more errors.